native_library/android/FirebaseANE/GoogleSignIn/src/main/java/com/tuarua/google/googlesignin/KotlinController.kt

```diff
@@ -18,41 +18,82 @@ package com.tuarua.google.googlesignin
 import android.content.res.Resources
 import com.adobe.fre.FREContext
 import com.adobe.fre.FREObject
+import com.adobe.fre.FREArray
 import com.google.android.gms.auth.api.signin.GoogleSignIn
+import com.google.android.gms.auth.api.signin.GoogleSignInClient
+import com.google.android.gms.auth.api.signin.GoogleSignInOptions
+import com.google.android.gms.common.api.Scope
 import com.tuarua.frekotlin.*
 import java.util.*
-import com.google.android.gms.auth.api.signin.GoogleSignInOptions
-import com.google.android.gms.auth.api.signin.GoogleSignInClient
 
 @Suppress("unused", "UNUSED_PARAMETER", "UNCHECKED_CAST", "PrivatePropertyName")
 class KotlinController : FreKotlinMainController {
+
     private var googleSignInClient: GoogleSignInClient? = null
+
+    override val TAG: String?
```

Comment:
This block should remain at bottom

Reply:
Are there any guidelines on how class members be arranged?